### ISSUE TITLE
characters are omitted that are accents in other languages

### ISSUE THREAD

**Issue by ericdq** [masonasons/Quinter#29]
When selecting the Remove emojis and other unicode characters from display names option, those accented characters are omitted, for example:

from
é (130)

to
Ñ (165)

I imagine that the same happens when selecting the Remove emojis and other unicode characters from tweet text option.

**Comment by trypsynth** [comment]
Yeah, it would. Those are unicode characters. Annoying, but seeing the current state of the Twitter API, I don't have a lot of drive to do the necessary research to fix it.
